alternative-settings.gradle

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
```

Comment:
Can we remove this file as well? looks like it's not needed anymore

Reply:
Not yet.